src/polygon.c

```diff
@@ -1478,3 +1479,53 @@ spherepoly_add_points_finalize(PG_FUNCTION_ARGS)
 	}
 	PG_RETURN_POINTER(poly);
 }
+
+
+Datum
+spherepoly_is_convex(PG_FUNCTION_ARGS)
+{
+	Vector3D	u,
+				v,
+				vsu,
+				wsv,
+				crs;
+	float8		cur = 0,
+				prev = 0;
```

Comment:
Do the variables on lines 1487-1493 line up vertically for you? They don't line up vertically in the diff output here on GitHub.com.

Reply:
Yes, they do.

Reply:
The default tab size on github is 8. It can be changes to 4 in Profile -> Settings -> Appearance -> Tab size... Not sure, how to change it for a specific repository and specific file type.

I changed it. For me it looks like this:

![image](https://github.com/postgrespro/pgsphere/assets/3510112/68769945-0c6f-42f2-9776-5981874645e5)